src/SpecBaseObject.php

```diff
@@ -484,4 +484,21 @@ public function getDocumentPosition(): ?JsonPointer
     {
         return $this->_jsonPointer;
     }
+
+    /**
+     * Handle extension properties with `x-` prefix.
+     * See https://github.com/OAI/OpenAPI-Specification/blob/3.0.2/versions/3.0.2.md#specificationExtensions
+     * @return array<string, mixed>
+     */
+    public function getExtensions(): array
+    {
+        $extensions = [];
+        foreach ($this->_properties as $propertyKey => $extension) {
+            if (mb_strpos($propertyKey, 'x-') !== 0) {
```

Comment:
why `mb_strpos` and not `strpos`? should be the same on characters `x-` as far as I see.

Reply:
it's a habit. I always use `mb_` functions in all kind of string searches. But yeah, i don't think it's really required in this case.

Reply:
```suggestion
            if (strpos($propertyKey, 'x-') !== 0) {
```